apps/projects/templates/a4_candy_projects/project_detail.html

```diff
@@ -279,6 +279,26 @@ <h1 class="project-header__title">{{ module.name }}</h1>
                 aria-expanded="false">
                 <div class="container">
                     <div class="row">
+                        <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
+                          <h5>{{ insight|title|slice:':9' }}</h5>
+                        </div>
+
+                        <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
+                          <p> {{ insight_label }} </p>
+                        </div>
+                        <p>
+                        {% for label, value in counts %}
+                            <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
+                                {{ label|title }}: {{ value }}
+                            </div>
+                        {% endfor %}
+                        </p>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
+                          <h5>{{ result_title|title }}</h5>
+                        </div>
```

Comment:
@philli-m in this block, we need some styling for the design: https://app.zeplin.io/project/5b051f895f8bcd4920567e89/screen/649ae3b1b6b6da22f3a82842

When you fetch the branch, you need to run migrations. Then in the dasboard project > results, click the checkbox for the insight display. And then go to view the page. 
Now it looks like that:
![image](https://github.com/liqd/adhocracy-plus/assets/5695572/4f5f9e53-d4e8-4206-8d3e-ee8b4d49a081)